crates/bevy_app/src/app_builder.rs

```diff
@@ -215,6 +276,25 @@ impl AppBuilder {
     }
 
     /// Inserts a resource to the current [App] and overwrites any resource previously added of the same type.
+    ///
+    /// A resource in bevy represents globally unique data. The resources must be added to bevy application
```

Comment:
Nit: `The resources must be added to bevy application` -> `Resources must be added to Bevy Apps`